pylsp/python_lsp.py

```diff
@@ -442,13 +442,13 @@ def lint(self, doc_uri, is_saved):
         workspace = self._match_uri_to_workspace(doc_uri)
         document_object = workspace.documents.get(doc_uri, None)
         if isinstance(document_object, Document):
-            self._lint_text_document(doc_uri, workspace, is_saved=is_saved)
+            self._lint_text_document(doc_uri, workspace, is_saved, doc_version)
```

Comment:
I don't understand how this is supposed to work if the `doc_version` variable is not declared before being passed here.

Reply:
Whoops! I was a bit trigger happy on the commit and didn't spot it, sorry 😅